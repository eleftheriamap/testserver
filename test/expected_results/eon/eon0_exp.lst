
test/exp/test/gen/eon/eon0_exp.bin:     file format binary


Disassembly of section .data:

0000000000000000 <.data>:
   0:	9147c0e7 	add	x7, x7, #0x1f0, lsl #12
   4:	51100f5a 	sub	w26, w26, #0x403
   8:	cafa8cf0 	eon	x16, x7, x26, ror #35
   c:	8a000000 	and	x0, x0, x0
