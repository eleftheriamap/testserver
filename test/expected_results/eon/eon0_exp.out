Registers:
X00    = 0000000000000000
X01    = 0000000000000000
X02    = 0000000000000000
X03    = 0000000000000000
X04    = 0000000000000000
X05    = 0000000000000000
X06    = 0000000000000000
X07    = 00000000001f0000
X08    = 0000000000000000
X09    = 0000000000000000
X10    = 0000000000000000
X11    = 0000000000000000
X12    = 0000000000000000
X13    = 0000000000000000
X14    = 0000000000000000
X15    = 0000000000000000
X16    = e00000805fe0ffff
X17    = 0000000000000000
X18    = 0000000000000000
X19    = 0000000000000000
X20    = 0000000000000000
X21    = 0000000000000000
X22    = 0000000000000000
X23    = 0000000000000000
X24    = 0000000000000000
X25    = 0000000000000000
X26    = 00000000fffffbfd
X27    = 0000000000000000
X28    = 0000000000000000
X29    = 0000000000000000
X30    = 0000000000000000
PC     = 000000000000000c
PSTATE : -Z--
Non-zero memory:
0x00000000 : 0x9147c0e7
0x00000004 : 0x51100f5a
0x00000008 : 0xcafa8cf0
0x0000000c : 0x8a000000
